charters/UploadService/charter.md

```diff
@@ -0,0 +1,80 @@
+# Upload Service
+
+## Description
+The Human Cell Atlas (HCA) Data Coordination Platform (DCP) Upload Service provides a file staging and validation facility. Upload Areas are created and deleted using a [REST](https://en.wikipedia.org/wiki/Representational_state_transfer) Application Programming Interface (API), which is secured so only the DCP Ingestion Service may use it. It stages files into cloud storage and computes checksums for the files. The validation service can run arbitrary validation jobs against files.
```

Comment:
what do you mean by `arbitrary validation`?

Reply:
I will use the term `user-defined` as suggested by @parthshahva above.